lib/rules/no-standalone-reverse-solidus.ts

```diff
@@ -0,0 +1,45 @@
+import type { RegExpVisitor } from "regexpp/visitor"
+import type { RegExpContext } from "../utils"
+import { createRule, defineRegexpVisitor, CP_BACK_SLASH } from "../utils"
+
+export default createRule("no-standalone-reverse-solidus", {
+    meta: {
+        docs: {
+            description: "disallow standalone reverse solidus (`\\`)",
```

Comment:
This rule should probably be renamed. I had no idea what a "reverse-solidus" was until now. "Backslash" is more commonly used.

Reply:
I personally liked `backslash`, but the name in the spec was `reverse-solidus`. I didn't know which one was easier for people to understand, so I followed the spec.
But you and I think backslash is good so change to `backslash`!